lego/apps/events/constants.py

```diff
@@ -18,6 +18,12 @@
     (EVENT, EVENT),
 )
 
+NORMAL = "normal"
+INFINITY = "infinity"
```

Comment:
Feel like `INFINITE` fits better, as it is an "infinite pool" not "infinity pool"